packages/date-time-utilities/src/dateGrid/getDayGrid.ts

```diff
@@ -54,14 +54,18 @@ export const getDayGrid = (options: IDayGridOptions): IDay[][] => {
   // in work week view if the days aren't contiguous we use week view instead
   const selectedDateRangeType = getDateRangeTypeToUse(dateRangeType, workWeekDays, firstDayOfWeek);
 
-  let selectedDates = getDateRangeArray(
-    selectedDate,
-    selectedDateRangeType,
-    firstDayOfWeek,
-    workWeekDays,
-    daysToSelectInDayView,
-  );
-  selectedDates = getBoundedDateRange(selectedDates, minDate, maxDate);
+  let selectedDates: Date[] = [];
+
+  if (!!selectedDate) {
+    selectedDates = getDateRangeArray(
+      selectedDate,
+      selectedDateRangeType,
+      firstDayOfWeek,
+      workWeekDays,
+      daysToSelectInDayView,
+    );
+    selectedDates = getBoundedDateRange(selectedDates, minDate, maxDate);
+  }
```

Comment:
I suggest to add a unit test for this case

Reply:
Added